test/log.spec.js

```diff
@@ -394,7 +401,7 @@ Object.keys(testAPIs).forEach((IPFS) => {
             id: 'X',
             heads: ['zdpuB2NAQ7cSh9MAfY91QC6Va56pQMJBXBaLoS6uQ1qNxqija']
           }
-          let log = new Log(ipfs, testACL, testIdentity, 'X')
+          let log = new Log(ipfs, testACL, testIdentity, { logId: 'X' })
           await log.append('one')
           const multihash = await log.toMultihash()
           const res = await Log.fromCID(ipfs, testACL, testIdentity, multihash, -1)
```

Comment:
Should the `-1` after multihash be in the passed as `{ length: -1 }`?

Reply:
Fixed!